configs/accengage.json

```diff
@@ -1,30 +1,41 @@
 {
   "index_name": "accengage",
   "start_urls": [
-    "https://docs-alpha.accengage.com"
+    "https://documentation.accengage.com",
+    "https://documentation.accengage.com/sdk",
+    "https://documentation.accengage.com/api",
+    "https://documentation.accengage.com/dataflow/introduction",
+    "https://documentation.accengage.com/user-guide/en",
+    "https://documentation.accengage.com/user-guide/fr",
+    {
+      "url": "https://documentation.accengage.com/faq",
+      "selectors_key": "faq"
+    }
   ],
-  "stop_urls": [],
+  "stop_urls": ["https://documentation.accengage.com/admin"],
   "selectors": {
-    "lvl0": {
-      "selector": "(//nav//a[contains(@class, 'active')])[1]/../../preceding::a[1]",
-      "type": "xpath",
-      "global": true,
-      "default_value": "Documentation"
+    "default": {
+      "lvl0": "h1, #title-text",
+      "lvl1": "h2",
+      "lvl2": "h3",
+      "lvl3": "h4",
+      "lvl4": "h5",
+      "text": "p, li, ul, ol"
```

Comment:
Please add only atomic tags like `p` and `li` tags (scoped into the main element as reviewed before). `ul` and `ol` are too broad and not oriented for textual content